package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitgo-utxo-lib",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Still not a bugfix but a new feature, so should at least bump minor, not patch. ;)

Reply:
Actually, I still think it should be the major, because downgrading from 1.6.1 to 1.6.0 would break Zcash.

Reply:
Using 1.7.0